src/main/groovy/jenkins/automation/builders/PipelineJobBuilder.groovy

```diff
@@ -53,8 +53,9 @@ class PipelineJobBuilder {
                 def stagesStr = stages.collect {
                     """
                         stage("${it.stageName}") {
-                            build job: "${it.jobName}"
-                            ${it.parameters ? 'parameters: ' + it.parameters : ''}
+                            build job: "${it.jobName}"${
+                                it.parameters ? ', parameters: ' + it.parameters : ''
```

Comment:
note that `inspect()` is not required here because params are being passed in as an already-escaped string (for better or worse)

Reply:
Does this preceding comma work?

Reply:
@champain can you explain your question a bit more? are you asking if I tested this?

Reply:
Ok, gotcha offline. Makes more sense when I see the inputs.

Reply:
👍